src/server/on-off-farm/controller.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * The first question to start the journey.
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const onOffFarmGetController = {
+  handler(req, h) {
+    req.yar.set('animal', 'cow')
+    const onOffFarm = req.yar.get('onOffFarm')
+
+    return h.view('on-off-farm/index', {
+      pageTitle: 'Are you moving the cattle on or off your farm?',
+      heading: 'Are you moving the cattle on or off your farm?',
```

Comment:
I don't think we're using these - looks like they're hardcoded later on?

And for what it's worth, I think it makes sense that they are hardcoded in the template.  Unless we extract all question-specific elements & have a fully generic 'radio.njk' template, then we may as well have all of the question specific text in the template itself?

That is, of course, except for dynamic elements like errors or currently selected values.

Reply:
the higher level template (layout.njk i think its called) is using them